source/docs/essentials/installation.md

```diff
@@ -0,0 +1,170 @@
+React application to your customers. In order to run it, you need to make sure
+you have `npm` (>= 5.x) and `node` (>= 8.x) installed on your machine.
+
+You can check your versions by running these commands:
+
+```bash
+npm -v
+node -v
+```
+
+If you don't have the minimum requirements,
+[please follow the instructions on Node.js website](https://nodejs.org/).
+
+<blockquote class="info">
+[**Front-Commerce Lite**](https://github.com/front-commerce/front-commerce-lite)
+is an alternate open-source project we created to let you start without any
+further requirements.
+
+**We don't want you to trust us blindly!** Because of this, we created a lighter
+version of Front-Commerce. It should make you feel how it is like to work with
+our product.
+
+While it isn't
+[fully featured](https://github.com/front-commerce/front-commerce-lite#what-it-is-not),
+we believe that if you enjoy working with it, you will enjoy working with
+Front-Commerce since we use the same concepts and the same file structure. You
+could even reuse most of your code within Front-Commerce if you want to upgrade
+to the licensed version. Please visit
+[Front-Commerce Lite project page on Github](https://github.com/front-commerce/front-commerce-lite)
+to get started with it.
+
+</blockquote>
+
+## Installation using the skeleton
+
+**This is the recommended way to start a new Front-Commerce project.**
+
+Front-Commerce skeleton repository provides a quick way to get started with
+Front-Commerce with sane default configurations.
+
+### Get the code
+
+In order to install the skeleton, you need to follow these instructions:
+
+```bash
+git clone --depth=1 git@gitlab.com:front-commerce/front-commerce-skeleton.git
+cd front-commerce-skeleton
+npm install
+```
+
+It may take a while…
+
+### Configure the project
+
+This skeleton comes with default values for most of the configuration. Below are
+the configurations you will most likely have to change/create.
+
+#### Configure the instance
+
+Front-Commerce uses environment variables for runtime configurations depending
+on the instance that runs (local, staging, production).
+
+Copy the `.env.dist` sample file to a local `.env` and adapt configurations.
+
+#### Configure stores
+
+Configure the project by creating a `src/config/stores.js` file. It should be an
+object with stores codes as keys and configurations as values.
+
+Here is an example:
+
+```js
+module.exports = {
+  default: {
+    locale: "en-GB",
+    currency: "EUR",
+    default_country_id: "GB",
+    localeData: require("react-intl/locale-data/en"),
+    countries: IsoCountries =>
+      IsoCountries.registerLocale(require("i18n-iso-countries/langs/en.json"))
+  }
+};
+```
+
+<blockquote class="info">
+**Magento:** when using Front-Commerce with Magento, store codes
+must match Magento ones.
+</blockquote>
+
+#### Configure remote services
+
+In order for Front-Commerce to interact with remote headless services, you must
```

Comment:
Instead of saying that, I'd add a bit of context. Not sure if it's better (it may be less actionable for the user).

> Front-Commerce can interact with remote headless services. But for security reasons, most services uses some tokens to ensure that you are the only with access to their data. These kind of tokens are configurable within the `src/config/serviceKeys/` folder.
>
> For instance, to interact with Magento2...

Reply:
This is a great idea!